### PR TITLE
Queue

### PR DESCRIPTION
**Pull request by pedrofelipe**
screen shot 2016-10-28 at 11 18 20 am
screen shot 2016-10-28 at 11 18 26 am
screen shot 2016-10-28 at 11 18 28 am

### CODE REVIEW

**Comment by gabrielrinaldi** [comment]
Ideally the side menu collapses

**Comment by pedrofelipe** [comment]
@gabrielrinaldi but it does! on mobile, at least. I attached new screenshots, what do you think?

File: app/scripts/card.js [hunk 1]
```diff
@@ -1,12 +1,52 @@
...
+                }
+            },
+            methods: {
+                toggle: function (key, index) {
+
+                    this.isExpanded = !this.isExpanded;
+                },
+                retry: function(e) {
+
+                    console.log(e);
```
Comment (viniciusalmeida) [inline app/scripts/card.js]: Do you will remove this `console.log`s soon?

In the future don't commit this kind of thing. Commit this callbacks with a empty function is better.

**Review by viniciusalmeida — APPROVED** [review]

File: app/scripts/card.js (hunk 1 above)
Comment (pedrofelipe) [reply app/scripts/card.js]: @viniciusalmeida removed!

**Pull request merged**